.github/workflows/ci.yml

```diff
@@ -64,12 +66,11 @@ jobs:
       - name: Rustdoc
         run: cargo doc --document-private-items --no-deps
 
-  c_library:
-    name: Rust tests and C library build
+  rust_tests:
+    name: Rust tests
```

Comment:
does it make sense to make this depend on `lint_rust` first?  Or does that just slow things needlessly down?  Or did you even do this on purpose so you can get test to run even if you have lint warnings?